connect4.hs

```diff
@@ -0,0 +1,62 @@
+module Connect4 where
```

Comment:
can we rename to `Connect4.hs`.
It doesn't work unless it is named that (for linux)